docs/modules/integrations/pages/xray-exporter.adoc

```diff
@@ -34,6 +34,10 @@ Please make sure that the `bdd.configuration.formats` property includes JSON val
 |false
 |Statuses of test cases allowed to update
 
+|`xray-exporter.assignee`
```

Comment:
`assignee` should follow after `project-key`  since they belong to a common area